armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1299,3 +1329,115 @@ def blockCollectionFactory(xsSettings, allNuclidesInProblem):
     return BLOCK_COLLECTIONS[blockRepresentation](
         allNuclidesInProblem, validBlockTypes=validBlockTypes
     )
+
+
+class LatticePhysicsUpdateFrequency(IntEnum):
```

Comment:
It seems possible that you could eliminate the word `Update` from this class name entirely and the meaning would still be clear. If you agree, that could help to reduce clutter where this is used, since it is pretty frequent that you type out `LatticePhysicsUpdateFrequency` and `black` forces you onto multiple lines.